src/haxelib/client/Main.hx

```diff
@@ -145,6 +145,7 @@ class Main {
 		apiVersion : "3.0",
 	};
 	static var IS_WINDOWS = (Sys.systemName() == "Windows");
+	static var PATH_SEPARATOR = if (IS_WINDOWS) ";" else ":";
```

Comment:
It's the platform specific delimiter that is used in `PATH` (among other things), see https://nodejs.org/api/path.html#path_path_delimiter.